src/overlays/actors/ovl_En_Kz/z_en_kz.c

```diff
@@ -71,7 +71,7 @@ u16 EnKz_GetTextNoMaskAdult(GlobalContext* globalCtx, EnKz* this) {
 
     if (INV_CONTENT(ITEM_TRADE_ADULT) >= ITEM_FROG) {
         if (!(gSaveContext.infTable[19] & 0x200)) {
-            if (CHECK_OWNED_EQUIP(EQUIP_TUNIC, 2)) {
+            if (CHECK_OWNED_EQUIP(EQUIP_TUNIC, PLAYER_TUNIC_ZORA)) {
```

Comment:
For the same reason, the player enum isn't supposed to be used here.
Also note that it's a bit confusing right now due to the name of its arguments but `CHECK_OWNED_EQUIP` doesn't actually take an "equip value". It takes an index of the equipment to check for, with 0 being the first one (so 0 for kokiri tunic, etc). Whereas, for instance, `CUR_EQUIP_VALUE` returns 0 for none equipped (1 for kokiri tunic, etc).